### PR TITLE
Moved scripts to start 36 robots from experimental branch

### PR DESCRIPTION
**Pull request by michalpelka**
The PR is cherry-picking (and fixing) the experimental branch used to record demo video:
https://github.com/michalpelka/ROSCon2023Demo/tree/mp/iron_video

### CODE REVIEW

File: ros2_ws/src/otto_fleet_nav/launch/otto_nav_launch.py [hunk 1]
```diff
@@ -156,7 +156,7 @@ def generate_launch_description():
 
     ld.add_action(rviz_cmd)
     ld.add_action(bringup_cmd)
-    ld.add_action(spawner)
+    # ld.add_action(spawner)
```
Comment (jhanca-robotecai) [inline ros2_ws/src/otto_fleet_nav/launch/otto_nav_launch.py:159]: Is this change needed? How does it affect the demo in a small scene?

**Review by jhanca-robotecai — COMMENTED** [review]
I have not run the code yet, but few question marks are there already.

File: ros2_ws/src/roscon2023_demo/bash/_navigate_deliberation.sh [hunk 2]
```diff
@@ -0,0 +1,15 @@
+#! /bin/sh
+
+SCRIPT=$(readlink -f $0)
+SCRIPTPATH=`dirname $SCRIPT`
+
+export ROBOT_NAMESPACE=$1
+export LINE=$2
+export TASK=$3
+echo Staring robot $ROBOT_NAMESPACE
+# kill -9 $(ps -aux | grep $ROBOT_NAMESPACE | awk '{print $2}')
```
[the remark this answers is not in the document]
Comment (jhanca-robotecai) [reply ros2_ws/src/roscon2023_demo/bash/_navigate_deliberation.sh:10]: My bad, You do not _echo_ the killing, but start the calls. Anyways: is the killing needed? It is commented out, so I assume you did not use it at the end.

File: ros2_ws/src/otto_fleet_nav/launch/otto_nav_launch.py (hunk 1 above)
Comment (michalpelka) [reply ros2_ws/src/otto_fleet_nav/launch/otto_nav_launch.py:159]: This line is not needed and should be removed.

**Review by jhanca-robotecai — APPROVED** [review]
All my remarks were answered.
